UI/js-src/lsmb/menus/MenuRestStore.js

```diff
@@ -0,0 +1,119 @@
+define(["dojo/_base/declare",
+    "dojo/store/JsonRest", "dojo/store/Observable",
+    "dojo/store/Memory", "lsmb/menus/Cache",
+    "dijit/Tree", "dijit/tree/ObjectStoreModel",
+    "dijit/Menu", "dijit/MenuSeparator", "dijit/PopupMenuItem",
+    "dojo/when", "dojo/dom", "dojo/ready"
+], function(declare, JsonRest, Observable,
+    Memory, lsmbStoreCache,
```

Comment:
the `lsmbStoreCache` variable doesn't follow the naming of the other variables. That's confusing.